tensorflow/tools/docker/Dockerfile.devel-gpu

```diff
@@ -78,12 +78,12 @@ WORKDIR /tensorflow
 
 # Configure the build for our CUDA configuration.
 ENV CI_BUILD_PYTHON python
-ENV LD_LIBRARY_PATH /usr/local/cuda/extras/CUPTI/lib64:$LD_LIBRARY_PATH
+ENV LD_LIBRARY_PATH /usr/local/cuda/lib64:$LD_LIBRARY_PATH
 ENV TF_NEED_CUDA 1
 ENV TF_CUDA_COMPUTE_CAPABILITIES=3.0,3.5,5.2,6.0,6.1
 
 RUN tensorflow/tools/ci_build/builds/configured GPU \
-    bazel build -c opt --config=cuda --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" \
+    bazel build -c opt --config=cuda action_env=LD_LIBRARY_PATH=${LD_LIBRARY_PATH} --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" \
```

Comment:
"--action_env" rather than "action_env"

Reply:
Whoops, typo. I'll fix it and test.

Reply:
http://ci.tensorflow.org/view/Experimental/job/experimental-docker-gpu/3/